packages/lisk-p2p/src/peer.ts

```diff
@@ -12,59 +12,158 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-/* tslint:disable:interface-name variable-name */
-import { PeerState } from './p2p_types';
 
-// TODO: Use to create outbound socket connection inside peer object.
-// TODO: const socketClusterClient = require('socketcluster-client');
+import {
+	P2PRequestPacket,
+	P2PMessagePacket,
+	P2PResponsePacket,
+	P2PNodeStatus,
+} from './p2p_types';
+
+import socketClusterClient from 'socketcluster-client';
 
 export interface PeerConfig {
 	readonly clock?: Date;
-	readonly height: number;
+	readonly height?: number;
```

Comment:
why height is optional now?

Reply:
yeah maybe we can default it to one according to the core, I will also change it in my PR.